docs/topics/proc_running-the-containerless-mta-cli.adoc

```diff
@@ -0,0 +1,54 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2024-11-15
+:_mod-docs-content-type: PROCEDURE
+
+[id="running-the-containerless-mta-cli_{context}"]
+= Running the containerless CLI
+
+You can perform an application analysis for Java applications by using the {ProductShortName} CLI that does not require installation of a container runtime. 
+
+.Prerequisites
+
+* You have OpenJDK version 17 or later installed on your system.
+* You have Maven installed on your system.
+* You have the `python3` package installed on your system.
+* You have the `JAVA_HOME` environmental variable set.
```

Comment:
```suggestion
* You have the `JAVA_HOME` environmental variable set.
* Optional: You defined the ` JVM_MAX_MEM`  system variable.
+
NOTE: Defining ` JVM_MAX_MEM` is recommended. If you do not have this variable set, the analysis might hang.
```

Hi @ibragins, please check this suggestion ^ Based on what I can see in the [ticket](https://issues.redhat.com/browse/MTA-4267), this prereq is not mandatory. So shall we have it as optional and just add a not that it is recommended? Or shall I make it just a usuall mandatory prereq? Thanks!

Reply:
@mpershina it depends on the way how Java was installed. If it was installed using installer or package manager - it will work without setting JAVA_HOME, but if user will just unpack a tarball - system should be informed somehow where is java binary. In this case this var should be defined manually.

Reply:
@ibragins, I'm just a little bit confused. Are we speaking about only setting the `JAVA_HOME` var? Or are you talking about both `JAVA_HOME` and `JVM_MAX_MEM`?  Sorry, I just need to clarify this for myslef to produce some great text, haha :-D

Reply:
@mpershina ,
 Optional: You defined the `JVM_MAX_MEM` ->  Optional: You **can define** the `JVM_MAX_MEM`